.gitignore

```diff
@@ -4,6 +4,7 @@
 /node_modules
 /.pnp
 .pnp.js
+package-lock.json
```

Comment:
@shades-7 This can also be removed. 
I think you used `npm` instead of `yarn` (mentioned in README.md of this repo) to install required dependencies and hence `package-lock.json` got created.

Reply:
yes i use npm